search-processors/src/main/java/org/opensearch/searchpipelines/questionanswering/generative/GenerativeQAResponseProcessor.java

```diff
@@ -126,6 +126,11 @@ public void processResponseAsync(
         }
 
         GenerativeQAParameters params = GenerativeQAParamUtil.getGenerativeQAParameters(request);
+        if (params == null) {
+            throw new IllegalArgumentException(
+                "generative_qa_parameters not found. Please provide ext.generative_qa_parameters to proceed."
```

Comment:
May be we can refer to this link: https://opensearch.org/docs/latest/search-plugins/conversational-search/#step-6-use-the-pipeline-for-rag

so that customer knows which values to provide in this parameter?